src/main/kotlin/com/pubnub/api/endpoints/objects/uuid/GetAllUUIDMetadata.kt

```diff
@@ -26,7 +26,7 @@ class GetAllUUIDMetadata internal constructor(
 
         return pubnub.retrofitManager.objectsService.getAllUUIDMetadata(
             subKey = pubnub.configuration.subscribeKey,
-            options = params + mapOf("include" to "type,status")
+            options = params
```

Comment:
What about modifying integration tests so that they check new parameter?
e.g. setGetAndRemoveChannelMetadata()

Reply:
How would you test new parameters in those tests?